test/IdRegistry/IdRegistry.t.sol

```diff
@@ -300,6 +300,45 @@ contract IdRegistryTest is IdRegistryTestSuite {
         assertEq(idRegistry.getRecoveryOf(1), address(0));
     }
 
+    /*//////////////////////////////////////////////////////////////
+                         VERIFY FID CUSTODY ADDRESS TESTS
+    //////////////////////////////////////////////////////////////*/
+
+    function testFuzzVerifyFidCustody(uint256 recipientPk, bytes32 digest) public {
+        uint256 deadline = _boundDeadline(0);
+        recipientPk = _boundPk(recipientPk);
+        address recipient = vm.addr(recipientPk);
+        bytes memory sig = _signRegister(recipientPk, recipient, address(0), deadline);
+
+        vm.prank(owner);
+        idRegistry.disableTrustedOnly();
+        idRegistry.registerFor(recipient, address(0), deadline, sig);
+        assertEq(idRegistry.getIdCounter(), 1);
+
+        bytes memory msg_sig = _signMsg(recipientPk, digest);
+        assertEq(idRegistry.verifyFidCustodyAddress(recipient, 1, digest, msg_sig), true);
+    }
+
+    function testFuzzCannotVerifyFidCustodyIfBadDigest(
```

Comment:
we should also have tests that verify failure if the fid is specified incorrectly, and potentially the custody address if we decide to retain tht parameter.

Reply:
I added 2 extra tests.